aptos_sdk/asymmetric_crypto_wrapper.py

```diff
@@ -163,16 +163,14 @@ def serialize(self, serializer: Serializer):
 
 class MultiSignature(asymmetric_crypto.Signature):
     signatures: List[Tuple[int, Signature]]
-    BITMAP_NUM_OF_BYTES: int = 4
+    MAX_SIGNATURES: int = 16
```

Comment:
I couldn't find this 16 signature limit?  Is this just for the SDK?

As far as I can tell, the limit is 255

https://github.com/aptos-labs/aptos-core/blob/main/types/src/transaction/authenticator.rs#L794-L798

Reply:
https://github.com/aptos-labs/aptos-core/blob/main/types/src/transaction/authenticator.rs#L811

distinct keys is 16... I think we kinda messed up here, but I think we can make this bigger without any impact since we use the bitvec.